apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -313,23 +303,26 @@ class VotePanelContent extends React.Component {
                     </VotingButton>
                   </ButtonsContainer>
                   {
-                    <Info.Action>
+                    <StyledInfo>
                       {user ? (
-                        <p>
-                          You will cast your vote with{' '}
-                          {userBalance === null
-                            ? '… tokens'
-                            : pluralize(userBalance, '$ token', '$ tokens')}
-                          , since it was your balance at the beginning of the
-                          vote {this.renderBlockLink(snapshotBlock)}.
-                        </p>
+                        <div>
+                          <p>
+                            You will cast your vote with{' '}
+                            {userBalance === null
+                              ? '… tokens'
+                              : pluralize(userBalance, '$ token', '$ tokens')}
+                            , since it was your balance at the beginning of the
```

Comment:
"it was your balance at the beginning of the vote created on 03/27/19 at 12:01 UTC"

Is quite the mouthful; perhaps we can revise this copy a bit @dizzypaty?

Suggestion:

"it was your balance when the vote was created (03/27/19 at 12:01 UTC)"